lib/Basic/Unix/TaskQueue.inc

```diff
@@ -107,9 +107,11 @@ public:
   /// \returns true on error, false on success
   bool execute();
 
-  /// \brief Reads data from the pipes, if any is available.
-  /// \returns true on error, false on success
-  bool readFromPipes();
+  /// \brief Reads data from the pipes, if any is available. If
+  /// \p untilEnd is true, reads until the end of the stream; otherwise
+  /// reads once (posslby with a retry on EINTR), and returns.
```

Comment:
Type: "posslby". Also, does this really belong in the "brief" description?

Reply:
fixed.